src/less/lightbox-dialog/lightbox-dialog.less

```diff
@@ -25,6 +25,27 @@
     .dialog-footer-content();
 }
 
+.lightbox-dialog__image {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-radius: var(--border-radius-100) var(--border-radius-100) 0 0;
+    height: 218px;
+    position: absolute;
+    width: 100%;
+}
+
+.lightbox-dialog__image + .lightbox-dialog__header > * {
+    margin-top: 218px;
+}
+
+.lightbox-dialog__image
```

Comment:
Oh, I just realized what you were saying about the complex selector. I had wrongfully assumed that we would have a modifier class on the lightbox similar to other modified implementations on lightbox (and other components). I didn't notice it missing before.

I think we should add that top-level modifier (`lightbox-dialog--expressive`) to this lightbox ; that's how we typically handle modified variations, even slightly modified variations. And then simply cascade down for override selectors like, `.lightbox-dialog--expressive .lightbox-dialog__close {}`.

Reply:
That is nicer, and actually required now that I know the padding actually also has to change. Thanks.